src/components/navigation/header/drawer/UserInfo.tsx

```diff
@@ -1,18 +1,21 @@
 import React, { FC } from 'react';
-import { ManagedSeller, MergedUser } from '@smg-automotive/auth';
+import { EnrichedSessionUser } from '@smg-automotive/auth';
 import { Box, Divider, GridItem, Stack } from '@chakra-ui/react';
 
 import { GarageIcon } from 'src/components/icons';
 import Avatar from 'src/components/avatar';
 
 type Props = {
-  user: MergedUser | null;
-  selectedTenant: ManagedSeller | null;
+  user: EnrichedSessionUser | null;
 };
 
-const DrawerUserInfo: FC<Props> = ({ user, selectedTenant }) => {
+const DrawerUserInfo: FC<Props> = ({ user }) => {
   if (!user) return null;
 
+  const selectedTenant = user.managedSellers?.find(
```

Comment:
Can we double check that for the case of impersonation, where the current seller will not be part of your managed sellers

Reply:
How can I test impersonation? Is it documented somewhere? Wouldn't it modify the token in a way where the `sellerId`s of the impersonated user are added to the token?

Reply:
Thanks for the ping, we checked in person and there is a guard in place